conda-store/pyproject.toml

```diff
@@ -65,10 +66,9 @@ conda-store = "conda_store.__main__:main"
 
 [tool.black]
 line-length = 88
-target-version = ['py38', 'py39', 'py310']
+target-version = ['py38', 'py39', 'py310', 'py311']
```

Comment:
I guess 3.12 probably makes sense to add here as well?

Reply:
+1 on adding it. The thing is that black "tries" to infer the versions, so I prefer explicitly having these per

> You should include all versions that your code supports. If you support Python 3.8 through 3.11, you should write:...